CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 - [S3 Repository] Add setting to control connection count for sync client ([#12028](https://github.com/opensearch-project/OpenSearch/pull/12028))
 - Views, simplify data access and manipulation by providing a virtual layer over one or more indices ([#11957](https://github.com/opensearch-project/OpenSearch/pull/11957))
 - Add Remote Store Migration Experimental flag and allow mixed mode clusters under same ([#11986](https://github.com/opensearch-project/OpenSearch/pull/11986))
+- Add Peer Recovery stats in node/cluster stats ([#12490](https://github.com/opensearch-project/OpenSearch/pull/12490))
```

Comment:
Planned to be shipped in 3.0 or 2.x? If 2.x, please move this to the 2x section

Reply:
Please also create a GH issue/PR on the docs repo for the respective documentation update and link it against the `Public documentation issue/PR created` checklist item in this PR's description